tests/target/issue-4746/crate.rs

```diff
@@ -0,0 +1,101 @@
+// rustfmt-imports_granularity: Crate
+
+// Long imports with reordering - from original issue
+use abaadfsasdfdsfdfas::aasdffjsioejr::abc::sdsdf::sdfsdfsdf::sdfsdfdsf::
+    asdfasdefasdasdfsdfdfasdf::asdfasdasedfafasdfasdf;
```

Comment:
I don't think this is formatting that's permitted based on the [formatting rules for large imports](https://github.com/rust-lang/fmt-rfcs/blob/master/guide/items.md#large-list-imports). I think we'd need to add braces in order to introduce this line break.